src/SybaseDB.js

```diff
@@ -35,7 +41,7 @@ function Sybase(host, port, dbname, username, password, logTiming, pathToJavaBri
 Sybase.prototype.connect = function(callback)
 {
     var that = this;
-    this.javaDB = spawn('java',["-jar",this.pathToJavaBridge, this.host, this.port, this.dbname, this.username, this.password]);
+    this.javaDB = spawn('java',["-jar",this.pathToJavaBridge, this.host, this.port, this.dbname, this.username, this.password, this.allowNull]);
```

Comment:
I am getting an error that says when allowNull is set to true
Error: Expecting the arguments: host, port, dbname, username, password